src/main/java/address/storage/StorageManager.java

```diff
@@ -181,16 +181,20 @@ public static UserPrefs getUserPrefs(File prefsFile) {
     public void start() {
         logger.info("Starting storage manager.");
         loadDataFromFile(prefs.getSaveLocation());
-        raise(new SaveLocationChangedEvent(prefs.getSaveLocation()));
```

Comment:
I removed this because this line doesn't fit here. i.e. there is no change to save location during startup.
If this cause regressions, it needs to be fixed without adding this back.